packages/@aws-cdk/core/lib/private/prepare-app.ts

```diff
@@ -100,6 +112,6 @@ function findCfnResources(root: IConstruct): CfnResource[] {
   return root.node.findAll().filter(CfnResource.isCfnResource);
 }
 
-interface INestedStackPrivateApi {
+interface IInternalStackPrivateApi {
   _prepareTemplateAsset(): boolean;
 }
```

Comment:
If you use my suggestion from above about `_synthesizeTemplate()`, pretty sure we'll be able to get rid of all these changes in `@aws-cdk/core`.

Reply:
this is gone now.